examples/multi-mig-http-lb/main.tf

```
@@ -67,6 +67,7 @@ module "cloud-nat-group2" {
   name       = "${var.network_prefix}-cloud-nat-group2"
 }
 
+# [START cloudloadbalancing_ext_http_gce]
 module "gce-lb-http" {
   source  = "../../"
```

For c.g.c. examples, we should source from the registry.


```suggestion
  source            = "GoogleCloudPlatform/lb-http/google"
  version           = "~> 5.1"
```

---

done